README.md

```diff
@@ -52,31 +70,34 @@ path as your binary, it will be autodetected and you could run by just calling i
 
 ### Global options
 
-- token: Offline token for authentication.
-- gateway-url: Gateway URL.
+- ocm-token: OCM Authorization token
+- ocm-token-url: Token URL (default "https://sso.redhat.com/auth/realms/redhat-external/protocol/openid-connect/token")
+- gateway-url: Gateway url to perform the test against (default "https://api.integration.openshift.com")
 - client:
   - id: OpenID client identifier.
   - secret: OpenID client secret.
-- duration-minutes: How long should the load test take.
 - output-path: Path to output results.
+- duration: Duration of each individual run in minutes. (default 1)
+- rate: Rate of the attack. Format example 5/s. (Available units 'ns', 'us', 'ms', 's', 'm', 'h') (default "1/s")
+- test-id: Unique ID to identify the test run. UUID is recommended (default "dc049b1d-92b4-420c-9eb7-34f30229ef46")
```

Comment:
May want to change this default to an actual uuid4 function call, so it generates a new UUID each time.